xAPI.md

```diff
@@ -1862,6 +1862,10 @@ The Experience API applies the concept of registration more broadly.  A registra
 considered to be an attempt, a session, or could span multiple Activities. There is no expectation that 
 completing an Activity ends a registration. Nor is a registration necessarily confined to a single Agent.
 
+The Registration id is also used when storing documents within the State resource, e.g. for 
```

Comment:
I agree with removing 'registration id' in all places for the reasons stated above. I agree with not having a SHOULD requirement since it's just sensible to do it but maybe that part needs stronger wording to get the point across...something like, "Naturally the same registration is used for requests to both the Statement and State resources relating to the same learning experience so all data captured for it is consistent."